resource/menu/client/cl_main_page.lua

```diff
@@ -63,42 +63,91 @@ if isMenuDebug then
     end)
 end
 
+
+
 local oldVehVelocity = 0.0
+
+---if ped is in any vehicle then delete
+local GetVehicle = function()
+    local ped = PlayerPedId()
+    local oldVeh = GetVehiclePedIsIn(ped, false)
+    if oldVeh and oldVeh > 0 then
+        oldVehVelocity = GetEntityVelocity(oldVeh)
+        DeleteVehicle(oldVeh)
+    end
+
+end
```

Comment:
We don't use anonymous function variable assignment anywhere else in the menu scripts. To stick with current conventions and code style, prefer named local function definitions.

```suggestion
local function getAndDeleteVehicle()
    local ped = PlayerPedId()
    local oldVeh = GetVehiclePedIsIn(ped, false)
    if oldVeh and oldVeh > 0 then
        oldVehVelocity = GetEntityVelocity(oldVeh)
        DeleteVehicle(oldVeh)
    end
end
```

Reply:
you right I should have follow the same way.